### ISSUE TITLE
Капельница и ИВЛ удаляют конечности скелетов.

### ISSUE THREAD

**Issue by pro100kotah** [TauCetiStation/TauCetiClassic#12295]
#### Подробное описание проблемы
При отрыве от ИВЛ, кардио-машины, капельницы и т.д. Игрок получает маленький брут на руку. В случае скелетов этот брут открывает конечность и отправляет ее в нуллспейс.
#### Что должно было произойти
Конечность, если и оторвалась от 5 брута, но выпала и ее можно присоединить.
#### Что произошло на самом деле
Исчезла. Пропала. Испарилась *jorekge*
#### Как повторить
Играть в игру
#### Дополнительная информация:
#61421

**Comment by Kiromaka** [comment]
Убрать возможность вставлять скелетам капельницу или добавить уроны проверку на расу. 
 Скелеты колятся, а спу нет :nedovolen:
 Добавьте good first issue нврн

**Comment by Slavik2001** [comment]
Возможно она не пропала, а получила такой спрайт у которого нету пикселей вовсе. Да и помимо скелетов, обычные конечности при повороте в определенную сторону пропадают

**Comment by pro100kotah** [comment]
> Возможно она не пропала, а получила такой спрайт у которого нету пикселей вовсе. Да и помимо скелетов, обычные конечности при повороте в определенную сторону пропадают с виду, но они физически есть, просто через ПКМ или альт-клик по полу

Нет. Я проверял прямо в раунде через альтклик и просто правым кликом по тайлу. Его не было нигде. Мы потратили на поиски приличное время и не нашли ее вообще нигде в маленькой комнате перед операционными. Конечность тупо делитнулась.
